ui/components/app/transaction-detail-item/transaction-detail-item.component.js

```diff
@@ -52,7 +52,9 @@ export default function TransactionDetailItem({
             color={COLORS.TEXT_DEFAULT}
             fontWeight={boldHeadings ? FONT_WEIGHT.BOLD : FONT_WEIGHT.NORMAL}
             variant={TYPOGRAPHY.H6}
-            margin={[1, 0, 1, 1]}
+            marginTop={1}
+            marginBottom={1}
+            marginLeft={1}
             boxProps={{ textAlign: TEXT_ALIGN.RIGHT }}
           >
             {detailTotal}
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props